rfc-005-template-contributing-process.md

```diff
@@ -0,0 +1,154 @@
+- **Before you start (optional)** - This optional guide is designed to help template users understand some of the key research, brainstorming, or pre-writing steps that they might need to do before adding this type of document to a guide. If this documentation type has certain key decisions that need to be made with stakeholders or if it has a dependency on some other type of document or process, that should be mentioned in this guide.
+- **Template example (optional)** - The Good Docs Project is in the process of designing a fake documentation project that can provide examples of our templates in action. Until this project is ready, template authors may optionally create an example of the template in action.
+
+
+## Links and prior art
+
+If you'd like to view the comment and revision history of these documents, you can see them in their Google Doc form:
+
+- [Template contribuing guide](https://docs.google.com/document/d/19pLi0JMlib11g5Di076hjyUM5KRkPHPvZLQpCwmPYi8/edit?usp=sharing)
+- [Template deliverables](https://docs.google.com/document/d/1uZPQ6jSj0TQoFSII_q6SbHgSySaGAZAevLvTuvXJeEU/edit?usp=sharing)
+- [Template roles and resources](https://docs.google.com/document/d/1fXp0Q7mv-VFj9GxScoXG1ZahsaAnkI3drPxuVNf9JVQ/edit?usp=sharing)
+
+
+## Open questions
+
+We haven't really found a good title for the template deliverable I'm currently calling "Before you start." Cameron prefers "theory." Suggestions are appreciated! Other ideas that were floated in the community:
+
+- Prewriting guide
+- Brainstorming guide
+- Template background
+- Template README
+
+
+## Decisions deferred
+
+Project management logistics that are out of scope for now include:
+- Whether template projects are represented as epics or issues
+- How template contributors indicate the status of their template set
+- How templates are planned for releases
+- Whether template changes are tracked and communicated at the release level or the template level
+
+Template formatting guidelines will also have to be deferred:
+- Template style guidelines
+- Markdown formatting guidelines
+- Metadata guidelines
+- Base template-related guidelines
+
+
+## Consequences
+
+Hopefully the consequence of this proposal will be that our community members have clear ways they can join a template working group and submit their first template to our project.
+
+
+## Feedback
+
+{Accepted suggestions will be incorporated into the proposal.
+Additional feedback or rejected suggestions will be noted here.}
+
+
+## Implementation checklist
+
+If this proposal is accepted, the following tasks must be completed:
+
+- [ ] Add these files to the `templates` repository and ensure they are linked from the website
+- [ ] Nominate a member to fill the templates coordinator role
+- [ ] Nominate members to fill the template mentor roles

```

Comment:
That's fair, Cameron. But I really can't see how we would operate without having clearly designated templates coordinator or template mentors in this process.

The same goes for pull request reviewers because those people will be given specific GitHub permissions that we won't freely distribute to everyone.

I can see the case for not defining community reviewers though. It can simply be anyone from the community.

Reply:
I'm going to note this concern in the feedback section of the RFC.